### ISSUE TITLE
Shortcuts that were disabled can't be assigned

### ISSUE THREAD

**Issue by Keith94** [kashiif/copy-urls-expert#27]
I tried to assign `Ctrl+Shift+C` as a shortcut for copying tabs, however it was conflicting with "Inspector" feature of Firefox. So I used [Keybinder](https://addons.mozilla.org/firefox/addon/keybinder) to disable the Inspector shortcut, but Copy Urls Expert still does not allow me to assign that shortcut (despite not being used for anything). Is there a way to fix that?

**Comment by kashiif** [comment]
Findings: Keybinder (and probably other such addons) adds `disabled=true` attribute to `<key>` when a shortcut is disabled. CUE should acknowledge `disabled` attribute in `<key>`.

**Comment by Keith94** [comment]
Nice findings. I noticed changing Inspector shortcut to something else (rather than disabling) also does not allow CUE to use `Ctrl+Shift+C` as a valid shortcut! Please check that too.

**Comment by kashiif** [comment]
> I noticed changing Inspector shortcut to something else (rather than disabling) also does not allow CUE to use `Ctrl+Shift+C` as a valid shortcut! Please check that too.

The way shortcuts work, you need to change the shortcut key in the same Firefox window where you invoke CUE options dialog. In other words, you cannot have two Firefox window, have CUE options dialog opened in one window and change the shortcut from another Firefox window.

**Comment by kashiif** [comment]
The original issue of disabled keys has now been fixed n ADVANCE_COPY branch. Can you please verify?

**Comment by Keith94** [comment]
Yes it seems to be fixed, but the CUE options now has odd warning text that the shortcut is already in use.

![firefox_2016-12-29_06-20-24](https://cloud.githubusercontent.com/assets/5490615/21545772/034f6aee-cd8f-11e6-9e85-172ec2ffa173.png)

This also means Keybinder allows me to change the shortcut directly in the new build, but CUE window still displays the previously assigned shortcut. Might simply be a bug.

![firefox_2016-12-29_06-32-06](https://cloud.githubusercontent.com/assets/5490615/21545987/961e2af8-cd90-11e6-9dba-4ba012fa21a0.png)

There may also be a problem where the new build doesn't allow saving any settings via 'save' button.

![firefox_2016-12-29_06-23-56](https://cloud.githubusercontent.com/assets/5490615/21545848/87d3d8c2-cd8f-11e6-83dd-e87cff8e7c12.png)

**Comment by kashiif** [comment]
> Yes it seems to be fixed, but the CUE options now has odd warning text that the shortcut is already in use.

Fixed now.

> This also means Keybinder allows me to change the shortcut directly in the new build, but CUE window still displays the previously assigned shortcut. Might simply be a bug.

Yeap, this is a bug. When user selects a shortcut in CUE options, it is saved into extension's preferences and loaded into Firefox window on every start. Obviously extensions like Keybinder do the same. It would be difficult to keep the shortcuts synchronized. I assume people using extensions like Keybinder clearly know what they are doing so they can fix it themselves.

TLDR; for managing CUE shortcuts, use CUE shortcuts tab in options (and not Keybinder).

> There may also be a problem where the new build doesn't allow saving any settings via 'save' button.

I could not reproduce it. There is [an existing issue on *nix operating systems](https://github.com/kashiif/copy-urls-expert/issues/20#issuecomment-216391140) but I can see you are on Windows. Can you confirm?

**Comment by Keith94** [comment]
Okay, regarding this issue, it now lets me assign a shortcut that was disabled (in this case Ctrl+Shift+C), but using that shortcut won't copy any urls.
